zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/utils/ProcessCacheUtil.java

```diff
@@ -68,4 +70,65 @@ public static List<String> extractCallActivityIdsFromDiagram(final ProcessEntity
   public static List<String> sortedCallActivityIds(final Collection<CallActivity> callActivities) {
     return callActivities.stream().map(BaseElement::getId).sorted().toList();
   }
+
```

Comment:
I left some comments in the RDBMS version of this file, but I think the same all applies here